### ISSUE TITLE
manage.py runcadencestrategies fails to send a valid observation_payload

### ISSUE THREAD

**Issue by griffin-h** [TOMToolkit/tom_base#420]
**Describe the bug**
`manage.py runcadencestrategies` fails to send a valid `observation_payload` to the LCO API when the observation record was created via the `LCOPhotometricSequenceForm`. Specifically, the instrument configuration is stored with each filter separately, e.g.:
```
ipdb> last_obs.parameters_as_dict
{'cadence_strategy': 'ResumeCadenceAfterFailureStrategy', 'ipp_value': 1.05, 'max_airmass': 2.0, 'facility': 'LCO', 'target_id': 1, 'observation_type': 'IMAGING', 'name': 'KIC 011606854', 'min_lunar_distance': 20, 'period': None, 'jitter': None, 'observation_mode': 'NORMAL', 'cadence_frequency': 5.0, 'proposal': 'LCO2021A-010', 'instrument_type': '1M0-SCICAM-SINISTRO', 'U': [], 'B': [], 'V': [], 'R': [], 'I': [], 'up': [], 'gp': [], 'rp': [120.0, 1, None], 'ip': [], 'zs': [], 'w': [], 'start': '2021-02-04T16:52:22', 'end': '2021-02-09T16:52:22'}
```
When I try to bind the `LCOPhotometricSequenceForm` to this data, the filter fields do not get populated:
```
ipdb> from custom_code.facilities.lco import LCOPhotometricSequenceForm
ipdb> form = LCOPhotometricSequenceForm(last_obs.parameters)
ipdb> for field in form.fields: print(field, form[field].value())
ipp_value 1.05
max_airmass 2.0
facility LCO
target_id 1
observation_type IMAGING
name KIC 011606854
min_lunar_distance 20
period None
jitter None
observation_mode NORMAL
cadence_frequency 5.0
proposal LCO2021A-010
instrument_type 1M0-SCICAM-SINISTRO
U [None, None, None]
B [None, None, None]
V [None, None, None]
R [None, None, None]
I [None, None, None]
u [None, None, None]
g [None, None, None]
r [None, None, None]
i [None, None, None]
z [None, None, None]
w [None, None, None]
cadence_strategy ResumeCadenceAfterFailureStrategy
```
This means that `form.observation_payload()` returns a blank `'instrument_configs'` field, which gets rejected by the LCO API with this error message:
```
ImproperCredentialsException: LCO: b'{"requests":[{"configurations":[{"instrument_configs":["A configuration must have at least one instrument configuration"]}]}]}'
```
(As a side bug, `ImproperCredentialsException` is a somewhat confusing name for the actual problem here.)

**To Reproduce**
1. Have an observation request in your database created using the `LCOPhotometricSequenceForm` that uses the `ResumeCadenceAfterFailureStrategy`.
2. Run `manage.py runcadencestrategies`.
3. See the `ImproperCredentialsException`.

**Expected behavior**
Somehow these exposure times and exposure counts need to be converted into the `'instrument_configs'` format, but I cannot figure out where this is supposed to happen. Once that is fixed, I believe the observation request should be submitted successfully.

**Desktop:**
- OS: macOS Big Sur
- Python Version: 3.7.3
- TOM Toolkit Version: 2.3.0

**Comment by dmcollom** [comment]
Yeah, this is a quirk with the MultiValueField--they're a little unusual to implement. Basically, the initialization data is coming into the form just fine, but the field isn't getting populated because it's a composite field. I'm taking a look at it, although it may take me a bit. And fair point on the ImproperCredentialsException--I'll roll that change into this one.

**Comment by dmcollom** [comment]
@griffin-h This should be fixed as of 2.4.2 (NOT 2.4.1, as I made a mistake and had to yank the version)--however, keep an eye on this. The fix definitely works, but I'm not thrilled about the MultiWidget stuff, and it's really poorly documented compared with most Django features. I'm pretty confident that this addresses our strange use case of reloading the form from the database, which is the cause of the problem, but if something else comes up, let me know and I'll address it ASAP.

Just to note, you probably need to reinitialize your cadence--this is something else we're shoring up, just not quite out yet.
